### ISSUE TITLE
Add "NetBox API" API

### ISSUE THREAD

**Issue by bcftv** [APIs-guru/openapi-directory#876]
**Format**: OpenAPI 3.0
**Official**: YES
**Url**: https://demo.netbox.dev/api/?format=json
**Name**: NetBox API
**Category**: enterprise
**Logo**:

**Comment by MikeRalphson** [comment]
Many thanks for the contribution. Please could you provide a link to the actual *machine-readable* API definition, in either OpenAPI or one of the other supported formats (RAML, API Blueprint, Postman etc)? Without this unfortunately we cannot add the API to the directory.

**Comment by MikeRalphson** [comment]
@bcftv - Closing due to inactivity - but please feel free to reopen the issue if necessary.

**Comment by bcftv** [comment]
Apologies - I missed the notifications on this.

Here's the OpenAPI JSON: https://demo.netbox.dev/api/docs/?format=openapi

And here's the browsable link: https://demo.netbox.dev/api/docs/

**Comment by MikeRalphson** [comment]
@bcftv - It looks like your API definition did not fully validate. Please mention (@) MikeRalphson to discuss. Here is the log output:

**Comment by MikeRalphson** [comment]
> openapi-directory@2.0.0 check
> node --no-warnings --insecure-http-parser ./node_modules/@Mermade/api-registry/registry.js check --host undefined https://demo.netbox.dev/api/docs/?format=openapi

https://demo.netbox.dev/api/docs/?format=openapi Ffetch failed
Saving metadata...
Exiting with 1

**Comment by MikeRalphson** [comment]
The endpoint seems very slow, hence I think our fetch is timing out.

**Comment by bcftv** [comment]
> The endpoint seems very slow, hence I think our fetch is timing out.

Yes - it seems pretty laggy for me too. I'm afraid that I don't have any control over where it's hosted.

I logged a feature request for them to host it somewhere that is not dependent on their demo server: (https://github.com/netbox-community/netbox/issues/11911)

**Comment by bcftv** [comment]
Looks like they [addressed this previously](https://github.com/netbox-community/netbox/issues/6423) and added some server-side caching but the docs are generated on demand so they can't generate a static file. 

If you can't adjust the timeout then I guess this is not going to be possible.

**Comment by MikeRalphson** [comment]
I'll have a look, thanks for your efforts!
